src/Checkbox.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component, forwardRef} from 'react';
+import styled from 'styled-components';
+import PropTypes from "prop-types";
+import {default as styledProps} from "@styled-system/prop-types";
+import {
+    border,
```

Comment:
Extra styled apis... where is the border used... ? 
Also background ?.. it is actually "The background utility includes style props for backgroundImage, backgroundSize, backgroundPosition, and backgroundRepeat."